builder/tart/builder.go

```diff
@@ -50,6 +51,9 @@ func (b *Builder) Prepare(raws ...interface{}) (generatedVars []string, warnings
 	err = config.Decode(&b.config, &config.DecodeOpts{
 		PluginType:  "packer.builder.tart",
 		Interpolate: true,
+		InterpolateFilter: &interpolate.RenderFilter{
+			Exclude: []string{"boot_command"},
```

Comment:
I am finding that when I tried adding `http_directory` it started ignoring my `boot_command` - could it be due to this?

Reply:
Answered in https://github.com/cirruslabs/packer-plugin-tart/issues/76#issuecomment-1588880660.